src/property-filter/internal.tsx

```diff
@@ -353,8 +365,7 @@ const PropertyFilterInternal = React.forwardRef(
                     }}
                     onRemoveToken={() => {
                       removeToken(tokenIndex);
-                      inputRef.current?.focus({ preventDropdown: true });
```

Comment:
No need to trigger the focusing here because it is handled by useListFocusController with outsideSelector.